Ops handbook hr management (#834)
publiccodenet/about: activities/hr-management/onboarding.md

## Onboarding for Operations Coordinator

The Operations Coordinator needs to add the new employee to Tentoo NMBRS:

* Add new employee by filling out “nieuwe medewerker” form. Once approved by Tentoo (can take a few days) the user account is created
* Pay attention when filling out the salary. At the moment their form doesn't accommodate for our salary structure (holiday allowance is calculated in the gross monthly salary)
* E.g. if someone will earn € 5500 gross and works 80%, the salary needs to be added as follows: 4400 / 108 * 100 = € 4074.07 gross excluding holiday allowance
* If the person works fulltime earning € 5500, the calculation is as follows: 5500 / 108 * 100 = € 5092,59 gross excluding holiday allowance
* To create the user account go to the profile of the new hire and click on 'home'
* On the bottom-right you see “medewerker login” there you can create the user account which can be used for both the desktop as the mobile NMBRS application. Pick scenario 2

## Onboarding for employee

> If the employee agrees to the terms and has no further questions, onboarding ensues as follows:

Reviewer: ```suggestion
If the employee agrees to the terms and has no further questions, onboarding takes place as follows:
```

* Contract is converted from draft to final version and uploaded to Docusign
* The Chief Executive, President of the board and the employee need to sign
* Once everyone has signed the contract, the fully signed version is sent to the employee by the Operations Coordinator

> * In the same email, the Operations Coordinator also send a welcome message and our onboarding package

Reviewer: ```suggestion
* In the same email, the Operations Coordinator also sends a welcome message and our onboarding package
```

### Welcome email and onboarding package

The welcome email and onboarding package consists of the following information:

* Letting the employee know that we have a BYOD (bring your own device) policy
* That we offer a monitor and good headset (also when working remotely). If the laptop breaks we can contribute to a new one
* An overview of accounts and tools they need and how they can access / activate these Links to About and other important resources a
* The onboarding package includes links to learning resources for GitHub, Markdown, links to inspirational talks from our board President, Ben Cerveny and other open source proponents, as well as links to open source organizations and communities

> * Welcome email template and onboarding package [are on our secret Drive.](https://drive.google.com/drive/folders/1KqeG3KxibxmbkhQl5yKUdsmTtMMnwlqB)

Reviewer: ```suggestion
* Welcome email template and onboarding package [are on our secret Drive](https://drive.google.com/drive/folders/1KqeG3KxibxmbkhQl5yKUdsmTtMMnwlqB).
```
